Godot_test/Game.cs

```diff
@@ -114,25 +118,31 @@ private Godot.Sprite2D AddChildFromScene(Tile tile, string scenePath)
 	if (scenePath == "res://Entities/Enemy.tscn")
 	{
 	  var en = tile as Enemy;
-	  var gototEn = instance.GetChild<GodotGame.Entities.Enemy>(0);
-	  gototEn.EnemyTile = en;
+	  var godotEn = instance.GetChild<GodotGame.Entities.Enemy>(0);
+		godotEn.EnemyTile = en;
+		godotEn.maxHp = en.Stats.Health;
 
 	  //entityDict[en] = instance.GetChild<GodotGame.Entities.Enemy>(0);
 	  //entityDict.Add(en, );
 	  var enemies = entityDict.Count;
 
-	  enemyList.Add(gototEn);
-	  if (ResourceLoader.Exists("res://Sprites/LivingEntities/Enemies/" + tile.tag1 + ".png"))
-	  {
-			spr.Texture = ResourceLoader.Load("res://Sprites/LivingEntities/Enemies/" + tile.tag1 + ".png") as Texture2D;
-	  }
-	  else
-			spr.Texture = ResourceLoader.Load("res://Sprites/LivingEntities/Enemies/Bat.png") as Texture2D;
+	  enemyList.Add(godotEn);
+			if (ResourceLoader.Exists(pathToEnemies + tile.tag1 + ".png"))
+			{
+				spr.Texture = LoadTexture(tile.tag1);
+			}
+			else
+				spr.Texture = LoadTexture("Bat");
 	}
 
 	return spr;
   }
 
+	private Texture2D LoadTexture(string path)
```

Comment:
Ponieważ ładuje to wrogów powinno sie nazywac :LoadEnemyTexture. Ale lepiej by bylo jakaś ogólnie to zrobić.
Tak żeby ta metoda widział ze jak wczytuje wroga to ma uzyć  pathToEnemies  a jak np InteractiveTile to scieżkę do InteractiveTiles